chalk/trail.py

```diff
@@ -15,10 +16,12 @@ class Trail(tx.Transformable):
     with illustrations/figures).
     """
 
-    def __init__(self, offsets: List[Vector]):
-        self.offsets = offsets
+    offsets: Vec2Array
+
+    def __init__(self, offsets: Vec2Array):
+        self.offsets = Vec2Array(offsets)
```

Comment:
If `offsets` are already of type `Vec2Array` then there is no need to wrap them again in `Vec2Array`.